__tests__/data/singular.json

```diff
@@ -1495,8 +1501,8 @@
           "affiliation": "Apple Store",
           "receipt_signature": "1234dfghnh",
           "purchase_transaction_id": "2134dfg",
-          "skan_last_call_timestamp": "1654532056",
-          "skan_first_call_timestamp": "1651853656",
+          "skan_last_call_timestamp": "2021-09-08T12:16:11.000Z",
+          "skan_first_call_timestamp": "2021-09-05T15:46:51.000Z",
```

Comment:
This is not in UNIX format

Reply:
This is the input payload, from which we will convert to UNIX format in our transformer and pass on to SIngular.